media-plugins/vidstab/files/vidstab-gentoo.diff

```diff
@@ -0,0 +1,31 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -25,7 +25,7 @@ else()
+     set(LIBSUFFIX "")
+ endif()
+ 
+-add_definitions( -Wall -O3 -g -Wno-pointer-sign -fPIC -std=gnu99)
++add_definitions( -Wall -Wno-pointer-sign -fPIC -std=gnu99)
```

Comment:
adding `-fPIC` is considered bad practice, as it kills performance on x86. Thi should be set by the user, not the build system. For setting C99, consider something more like https://cmake.org/cmake/help/v3.1/prop_tgt/C_STANDARD.html, which is better than hardcoding compiler-specific flags.